src/test/java/com/application/data/service/SkiResortServiceIntegrationTest.java

```diff
@@ -0,0 +1,541 @@
+package com.application.data.service;
```

Comment:
Frage: Brauchst du bei fast jedem Test ein neues Skigebiet? Wäre es nicht "einfacher" dies einmal anfangs mit z.B. @BeforeTest zu erzeugen und dann eines weiterzuverwenden?
Weiß aber eben nicht ob das was mit z.B. Mockito zu tun hat.

Reply:
Vollkommen gerechtfertig die Frage. 
Problem ist dass ich bei den Integration-Tests die Datenbankanbindung brauche. Weder mit @Before (vor jedem Test ausgeführt) noch mit @BeforeClass (nur bei Initierung ausgeführt) kann ich Umfänge in die Datenbank speichern, näheres dazu hier:
https://stackoverflow.com/questions/20295578/difference-between-before-beforeclass-beforeeach-and-beforeall

Was natürlich geht, was ich bei den Abgabe-Files (UserSerivceIntegrationTest, UserServiceUnitTest) auch gemacht habe, die Erstellung der Instanz in eine eigene Methode zu packen und lediglich die Namen anzupassen (z.B. User hat ein Unique Constraint und kann somit nicht nur einmal erzeugt werden).
Die anderen Files muss ich tatsächlich noch nachbearbeiten und den redundanten Code in eine eigene Methode packen